api/api_test.go

```diff
@@ -20,7 +20,7 @@ const (
 )
 
 func newTestSignHandler(t *testing.T) (h http.Handler) {
-	h, err := NewSignHandler(testCaFile, testCaKeyFile, "")
+	h, err := NewSignHandler(testCaFile, testCaKeyFile, "", nil)
```

Comment:
I understand the test needs a (mock) remote signer. Ignore the above comment. Let's address this in another PR.